tony-core/src/main/java/com/linkedin/tony/TonyConfigurationKeys.java

```diff
@@ -134,6 +134,10 @@ private TonyConfigurationKeys() {
   public static final String APPLICATION_DISTRIBUTED_MODE = TONY_APPLICATION_PREFIX + "distributed-mode";
   public static final String DEFAULT_APPLICATION_DISTRIBUTED_MODE = DistributedMode.GANG.name();
 
+  public static final String TONY_APPLICATION_MAPREDUCE_PREFIX = TONY_APPLICATION_PREFIX + "mapreduce.";
+  public static final String APPLICATION_MAPREDUCE_FRAMEWORK_PATH = TONY_APPLICATION_MAPREDUCE_PREFIX + "framework.path";
```

Comment:
i think we can scrap TONY_APPLICATION_MAPREDUCE_PREFIX and just make this TONY_APPLICATION_PREFIX + framework.path - we can rename the variable APPLICATION_FRAMEWORK_PATH too

Reply:
Yeah, I thought about it. But we have another configuration, "tony.application.framework" used to decide the deep learning framework, e.g., TensorFlow or PyTorch. I am wondering if they are too similar. (maybe not?) Will there be a better name?

Reply:
Makes sense. How about `tony.application.dependencies.location` and `tony.application.dependencies.classpath`? I'd prefer not to have mapreduce in the name, logically they should be separate

Reply:
I update the dependencies with hadoop. Please let me know if it works for you. Thanks!